### PR TITLE
v0.2.0 changelog

### PR DESCRIPTION
**Pull request by alfred-landrum**
High level summary of changes since v0.1.0.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## v0.2.0
+
+* Per-platform binaries will be available as Github release assets.
+* Documentation examples can be verified via `test make-heavy`.
```
Comment (mikesbrown) [inline CHANGELOG.md]: `make test-heavy`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## v0.2.0
+
+* Per-platform binaries will be available as Github release assets.
+* Documentation examples can be verified via `make test-heavy`.
```
Comment (mikesbrown) [inline CHANGELOG.md]: Sorry for not saying this in an earlier comment. If we want to nitpick, the "Documentation examples" are really the "zq examples under `zql/docs`" and nothing more. The examples in the top-level README are not verified, for example. You don't have to change this if it's not important.

**Comment by alfred-landrum** [comment]
@mikesbrown : I'm looking at the changes in master since v0.1.0, and selecting highlights.

**Comment by mikesbrown** [comment]
To answer my own question:

> Do you have a candidate Git hash for v0.2.0?

Notes look good from v0.1.0 to baecc6a.

**Pull request merged**